charts/plantuml/values.yaml

```diff
@@ -55,6 +55,9 @@ ingress:
   # @default -- See _values.yaml_
   hosts: []
   #   - plantuml.local
+  # -- (string) Ingress path.
+  # @default -- /
```

Comment:
```suggestion
```
`helm-docs` picks up the default from the values.

Reply:
Thanks for the feedback @stevehipwell . Updated the PR accordingly.